Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -117,7 +117,9 @@ constexpr BugInfo m_known_bugs[] = {
     {API_VULKAN, OS_ALL, VENDOR_ARM, DRIVER_ARM, Family::UNKNOWN, BUG_SLOW_CACHED_READBACK_MEMORY,
      -1.0, -1.0, true},
     {API_VULKAN, OS_ALL, VENDOR_QUALCOMM, DRIVER_QUALCOMM, Family::UNKNOWN,
-     BUG_SLOW_CACHED_READBACK_MEMORY, -1.0, -1.0, true}};
+     BUG_SLOW_CACHED_READBACK_MEMORY, -1.0, -1.0, true},
+    {API_VULKAN, OS_OSX, VENDOR_ALL, DRIVER_PORTABILITY, Family::UNKNOWN,
+     BUG_BROKEN_GPU_TEXTURE_DECODING, -1.0, -1.0, true}};
```

Comment:
Could you add a trailing comma? Helps make future diffs cleaner.

Reply:
Like that?